### PR TITLE
M kovalsky/alignment

### PR DESCRIPTION
**Pull request by m-kovalsky**

### CODE REVIEW

File: src/sempy_labs/_model_bpa_bulk.py [hunk 1]
```diff
@@ -87,94 +92,90 @@ def run_model_bpa_bulk(
...
-                        delta_table_name=output_table,
-                        write_mode="append",
-                        merge_schema=True,
-                    )
-                    print(
-                        f"{icons.green_dot} Saved BPA results to the '{output_table}' delta table."
-                    )
+                    except Exception:
+                        print(
+                            f"{icons.red_dot} Model BPA failed for the '{dataset_name}' semantic model within the '{wksp}' workspace."
```
Comment (eisber) [inline src/sempy_labs/_model_bpa_bulk.py:156]: why not print the exception message?

File: src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py [hunk 2]
```diff
@@ -101,168 +100,134 @@ def migrate_calc_tables_to_lakehouse(
...
-                                            " ", "_"
-                                        ).lower()
-
-                                        spark_df = spark.createDataFrame(df)
-                                        filePath = create_abfss_path(
-                                            lakehouse_id=lakehouse_id,
-                                            lakehouse_workspace_id=lakehouse_workspace_id,
-                                            delta_table_name=delta_table_name,
+                                    except Exception:
+                                        dataType = next(
```
Comment (eisber) [inline src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py:169]: what kind of exception do you expect here?

**Review by eisber — APPROVED** [review]

File: src/sempy_labs/_model_bpa_bulk.py (hunk 1 above)
Comment (m-kovalsky) [reply src/sempy_labs/_model_bpa_bulk.py:156]: updated to this:

except Exception as e:
                        print(
                            f"{icons.red_dot} Model BPA failed for the '{dataset_name}' semantic model within the '{wksp}' workspace."
                        )
                        print(e)

File: src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py (hunk 2 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py:169]: It's masking the Exception. I will update this in another PR with a better method.

File: src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py [hunk 3]
```diff
@@ -318,115 +283,106 @@ def migrate_field_parameters(
...
+                    row
+                    for row in rows
+                    if not any(
+                        value in row for value in dfC_CalcColumn["Column Object"].values
+                    )
+                ]
+                updated_query_string = "\n".join(filtered_rows)
+
+                # Remove extra comma
+                lines = updated_query_string.strip().split("\n")
```
[the remark this answers is not in the document]
Comment (m-kovalsky) [reply src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py:311]: It is finding any calculated columns and removing them from the dax expression.

**Pull request merged**